bin/templates/cordova/lib/prepare.js

```diff
@@ -89,6 +91,71 @@ module.exports.prepare = function (cordovaProject, options) {
     return this.parser.update_project(this.config, options);
 };
 
+/**
+ * Update Electron Splash Screen image.
+ */
+function updateSplashScreens (cordovaProject, config, locations) {
+    const splashScreens = cordovaProject.projectConfig.getSplashScreens('electron');
+
+    // Skip if there are no splash screens defined in config.xml
+    if (!splashScreens.length) {
+        events.emit('verbose', 'This app does not have splashScreens defined');
```

Comment:
```suggestion
        events.emit('verbose', 'This app does not have splash screens defined.');
```

This change is to create consistency between other platform's verbose messaging.